CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ WDL `size` engine function now works for HTTP files.
 Cromwell can now send logs to Azure Application Insights. To enable, set environment 
 variable `APPLICATIONINSIGHTS_INSTRUMENTATIONKEY` to your account's key. [See here for information.](https://learn.microsoft.com/en-us/azure/azure-monitor/app/sdk-connection-string)
 
+### Workflow Completion Callback
+
+Cromwell can be configured to send a POST request to a specified URL when a workflow completes. The request body
+includes the workflow id, terminal state, and (if workflow succeeded) final outputs. 
```

Comment:
Might be nice to include a link to the relevant section of the permanent docs.

Reply:
I thought of that but didn't want to link to a currently-nonexistent page. Probably a good idea to point people somehow, though, we can include the name so they can search it.